allensdk/brain_observatory/ecephys/ecephys_project_api/lims_api_mixin.py

```diff
@@ -0,0 +1,59 @@
+import psycopg2
```

Comment:
Refactor into internal/api, merge functionality with PostgresQueryMixin, check for regressions, reduce implementation redundancy